### PR TITLE
complement errors relating to lazy uniqueness check

### PR DESCRIPTION
**Pull request by ekexium**
Signed-off-by: ekexium <eke@fastmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

The PR complenets the changes happened in recent TiDB master: 
- The message of error 1062 has changed. 
- ~~PessimisticLockError can occur when users use the lazy uniqueness check feature inappropriately.~~

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: constraints.md [hunk 1]
```diff
@@ -262,9 +262,11 @@ ERROR 1062 (23000): Duplicate entry 'bill' for key 'username'
     ```
 
     ```
-    ERROR 8147 (23000): transaction aborted because lazy uniqueness check is enabled and an error occurred: [kv:1062]Duplicate entry 'bill' for key 'username'
+    ERROR 8147 (23000): transaction aborted because lazy uniqueness check is enabled and an error occurred: [kv:1062]Duplicate entry 'bill' for key 'users.username'
     ```
 
+- 关闭该变量时，1062 Duplicate entry 错误的语句所报的不一定是当前语句发生的错误，因此在一个事务操作多个表，且这些表有同名索引时，注意报错的是哪个表的哪个索引。
```
Comment (qiancai) [inline constraints.md:268]:
```suggestion
- 关闭该变量时，`1062` Duplicate entry 报错不一定是当前执行的 SQL 语句所发生的错误。因此，在一个事务操作多个表，且这些表有同名索引时，请注意 `1062` 报错信息中提示的是哪个表的哪个索引发生了错误。
```

**Review by sticnarf — APPROVED** [review]
LGTM

File: constraints.md [hunk 2]
```diff
@@ -262,9 +262,11 @@ ERROR 1062 (23000): Duplicate entry 'bill' for key 'username'
     ```
 
     ```
-    ERROR 8147 (23000): transaction aborted because lazy uniqueness check is enabled and an error occurred: [kv:1062]Duplicate entry 'bill' for key 'username'
+    ERROR 8147 (23000): transaction aborted because lazy uniqueness check is enabled and an error occurred: [kv:1062]Duplicate entry 'bill' for key 'users.username'
     ```
 
+- 关闭该变量时，`1062` Duplicate entry 报错不一定是当前执行的 SQL 语句所发生的错误。因此，在一个事务操作多个表，且这些表有同名索引时，请注意 `1062` 报错信息中提示的是哪个表的哪个索引发生了错误。
```
Comment (Oreoxmt) [inline constraints.md:268]:
```suggestion
- 关闭该变量时，`1062 Duplicate entry` 报错不一定是当前执行的 SQL 语句所发生的错误。因此，在一个事务操作多个表，且这些表有同名索引时，请注意 `1062` 报错信息中提示的是哪个表的哪个索引发生了错误。
```

Comment (Oreoxmt) [reply constraints.md:268]: 1062 是否需要加到[错误码与故障诊断](https://docs.pingcap.com/zh/tidb/stable/error-codes)文档中

Comment (ekexium) [reply constraints.md]: 我觉得原来既然没有那可以不加吧

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**